style.css

```diff
@@ -249,3 +260,80 @@ body {
   font-family: "Inter", sans-serif;
   font-size: 2.2rem;
 }
+
+/* ============= contact form start ============= */
+
+.section-contact-form {
+  padding: 7rem 0;
+  border-bottom: 1px solid #dfe1e6;
+}
```

Comment:
- [ ] Based on project requirements, kindly use flexbox to style this  section of the form.

## Requirement
![image](https://user-images.githubusercontent.com/50990925/235845574-c6ca7ff6-b15b-43ad-b78f-748b16dff455.png)